eslintrc.json

```diff
@@ -48,22 +49,23 @@
     "no-void": 2,
     "no-with": 2,
     "prefer-promise-reject-errors": 2,
-    "require-await": 2,
+    "require-await": 0,
     "vars-on-top": 2,
     "wrap-iife": 2,
     "no-delete-var": 2,
-    "no-shadow": 2,
+    "no-shadow": 0,
```

Comment:
This rule is useful especially when somebody that did not actually write the code, needs to read it for debugging or review cases.